.github/workflows/ci.yml

```
@@ -46,6 +46,34 @@ jobs:
       - run: cargo test
         env:
           RUSTFLAGS: ${{ env.RUSTFLAGS }} --cfg async_process_force_signal_backend
+  
+  test-android:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        rust: [nightly, beta, stable]
```

```suggestion
        rust: [nightly]
```

Little bit overkill otherwise